src/styles/blocks/card.scss

```diff
@@ -0,0 +1,72 @@
+.card {
+  box-sizing: border-box;
+  background-color: $color-white;
+  border: 1px solid $color-gray;
+  border-radius: 5px;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  padding:  32px 16px 16px;
+  display: flex;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+    display: flex;
+  }
+
+  &__name {
+    color: $color-black-main;
+    margin-top: 0;
```

Comment:
remove it in `index.scss` file or create a separate `reset.scss` file for reset default rules